applications/nrf_desktop/configuration/common/click.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef _CLICK_H_
+#define _CLICK_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct click_detector_config {
+	u8_t detector_id;
+	u16_t key_id;
```

Comment:
I think it would be better to pass array of keys to handle

Reply:
ok, will use single detector for multiple keys - based on one work